File: test_data/python/test_main/aas_core_meta.v3rc2/input/snippets/Verification/is_xs_date_time_stamp_utc.py
def is_xs_date_time_stamp_utc(
        value: str
) -> bool:
    """
    Check that :paramref:`value` is a ``xs:dateTimeStamp`` with
    the time zone set to UTC.
    """
    if matches_xs_date_time_stamp_utc(value) is None:
        return False

    date, _ = value.split('T')
    return is_xs_date(date)

File: test_data/typescript/test_main/aas_core_meta.v3rc2/input/snippets/Verification/is_xs_date_time_stamp_UTC.ts
/**
 * Check that `value` is a valid `xs:dateTimeStamp` with
 * the time zone set to UTC.
 *
 * @param value - to be checked
 * @returns `true` if `value` is a valid `xs:dateTimeStamp` with the UTC time zone
 */
export function isXsDateTimeStampUtc(
  value: string
): boolean {
  if (!matchesXsDateTimeStampUtc(value)) {
    return false;
  }

  const date = value.split("T", 1)[0];
  return isXsDate(date);
}
